compositions/terrajet-aws-provider/s3/s3-composition.yaml

```diff
@@ -0,0 +1,98 @@
+# Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+# SPDX-License-Identifier: MIT-0
+
+apiVersion: apiextensions.crossplane.io/v1
+kind: Composition
+metadata:
+  name: s3bucket.jet.awsblueprints.io
+  labels:
+    awsblueprints.io/provider: jet-aws
+    awsblueprints.io/environment: dev
+    s3.awsblueprints.io/configuration: standard
+    crossplane.io/xrd: xs3buckets.jet.awsblueprints.io
```

Comment:
Do we care about this?

Reply:
Not really but I kept it for consistency with other compositions